arch/arm64/boot/dts/freescale/imx8mm-crocodile-no-m4.dts

```diff
@@ -111,41 +110,41 @@
 
 	pinctrl_i2c4: i2c4grp {
 		fsl,pins = <
-			CROCODILE_IMX8_I2C_4_SCL	0x400001c3
-			CROCODILE_IMX8_I2C_4_SDA	0x400001c3
+			MX8MM_IOMUXC_I2C4_SCL_I2C4_SCL /*CROCODILE_IMX8_I2C_4_SCL*/	0x400001c3
+			MX8MM_IOMUXC_I2C4_SDA_I2C4_SDA /*CROCODILE_IMX8_I2C_4_SDA*/	0x400001c3
```

Comment:
Again, many thanks for aligning it to mainline :)

One nit, the comments should be after the pad settings, so:

`MX8MM_IOMUXC_I2C4_SCL_I2C4_SCL /*CROCODILE_IMX8_I2C_4_SCL*/	0x400001c3`

should be:

`MX8MM_IOMUXC_I2C4_SCL_I2C4_SCL	0x400001c3   /*CROCODILE_IMX8_I2C_4_SCL*/`

This applies to all.